gossip/src/crds_value.rs

```diff
@@ -494,27 +499,77 @@ impl Sanitize for NodeInstance {
 pub struct RestartLastVotedForkSlots {
     pub from: Pubkey,
     pub wallclock: u64,
-    pub slots: Vec<CompressedSlots>,
+    pub compressed_slots: Vec<u8>,
+    pub uncompressed_bytes: u64,
```

Comment:
what is uncompressed_bytes?

Reply:
When we uncompress we need to say how big the output buffer is. Of course we can reserve a constant sized buffer which is bigger enough, but I think this is good for debugging. Because you can use number of bytes to see how many slots are encoded.

Reply:
see my other comment about `uncompressed_bytes`.
I think we better remove it.